logstash.advisories.yaml

```diff
@@ -130,6 +130,10 @@ advisories:
             componentType: java-archive
             componentLocation: /usr/share/logstash/vendor/bundle/jruby/3.1.0/gems/logstash-input-http-3.8.0-java/vendor/jar-dependencies/io/netty/netty-codec-http/4.1.100.Final/netty-codec-http-4.1.100.Final.jar
             scanner: grype
+      - timestamp: 2024-04-11T18:43:49Z
+        type: true-positive-determination
```

Comment:
Let's make this a `pending-upstream-fix` since we're awaiting an upstream release!

Reply:
Okay